third-party/devtools-tests/e2e/lighthouse/navigation_test.ts

```diff
@@ -58,6 +58,9 @@ describe('Navigation', async function() {
           // TODO: Figure out why these are emitted in FR.
```

Comment:
What's the plan for this?

Reply:
So far these don't seem to impact the LH result at all 🤷 . I'm hoping it has something to do with Puppeteer attaching to more targets than LH needs, so maybe our [updates to the Puppeteer connection in DT](https://chromium-review.googlesource.com/c/devtools/devtools-frontend/+/3858690) will help.

Reply:
I looked into it briefly.

It appears that the sessionRouter in DT is trying to manage sessions. 

problem 1: the new sessions that puppeteer and LH (via pptr) are creating arent _known_ to the CDT frontend, so it's kinda confused. it's repeatedly getting messages these sessions but the `suppressUnknownMessageErrors` boolean mostly keeps it quiet.

problem 2: when the lighthouse_worker exits, we have some `Target.detachedFromTarget` events that hit the sessionRouter. It does some bookkeeping and removes a sessionId tracking that target. After this, a few more events (`Target.targetInfoChanged` and `Target.targetDestroyed`) come in for the now-orphaned session.  (Now, without the sessionId being tracked, the `suppressUnknownMessageErrors` boolean wont save us. I suspect that if we actively detach from some targets... it might help.


fixing problem 2: I only saw this happen with the lighthouse_worker target, but it's possible it happens for all oopif/worker targets that the page spawns.  it doesn't look like these errors are indicative of a problem.. IMO it's just highlighting that our targetmgmt across these 3 layers is not perfectly coordinated.

Reply:
That makes a lot of sense, thanks for looking into it! I'll open a p3 issue in DT to track this.